source/command/analyzeCommand.ml

```diff
@@ -324,11 +341,15 @@ let run_analyze analyze_configuration =
           | _ -> true)
         ~f:(fun scheduler ->
           with_performance_tracking ~debug ~f:(fun () ->
-              TaintAnalysis.run_taint_analysis
-                ~static_analysis_configuration
-                ~build_system
-                ~scheduler
-                ());
+              if static_analysis_configuration.verify_taint_config_only then
+                let _ = verify_taint_configuration ~static_analysis_configuration () in
+                ()
+              else
```

Comment:
We can do this earlier. Maybe after line 334?

Also, we can use `verify_taint_configuration ~static_analysis_configuration () |> ignore` to discard the result.

Reply:
Much better! Thanks for this suggestion :)